tools/base_labels/__init__.py

```
@@ -0,0 +1,83 @@
+    "properties_section_bounds",
+    "quote_yaml_scalar",
+    "rel",
+    "resolve_base",
+    "save_config",
+    "scan_base_labels",
+    "source_hash",
+    "strip_code_fences",
+    "translate_base_labels",
+    "unquote_yaml_scalar",
+    "update_config_sources",
+]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)